dask/dataframe/core.py

```diff
@@ -1662,6 +1666,19 @@ def meth(self, other, level=None, fill_value=None, axis=0):
         meth.__doc__ = op.__doc__
         bind_method(cls, name, meth)
 
+    @classmethod
+    def _bind_comparison_method(cls, name, comparison):
+        """ bind comparison method like DataFrame.add to this class """
+
+        def meth(self, other, level=None, fill_value=None, axis=0):
```

Comment:
Tracked this bug down. I implemented this keyword because I was looking at the pandas 19.0 docs but I've since realized dask is built on pandas 18.0.